MsCorePkg/Library/MacAddressEmulationPlatformLibNull/MacAddressEmulationPlatformLibNull.inf

```diff
@@ -0,0 +1,33 @@
+## @file
+#
+# NULL implementation of the platform library to provide hooks for the MAC Address Emulation feature.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION                    = 1.27
+  BASE_NAME                      = MacAddressEmulationPlatformLib
+  FILE_GUID                      = 75C79B3A-9D3A-44B5-8CA0-30ACBDF745CC
+  MODULE_TYPE                    = DXE_DRIVER
+  VERSION_STRING                 = 1.0
+  LIBRARY_CLASS                  = MacAddressEmulationPlatformLib
+
+[Sources]
+  MacAddressEmulationPlatformLibNull.c
+
+[Packages]
+  MdePkg/MdePkg.dec
+  MdeModulePkg/MdeModulePkg.dec
```

Comment:
i don't think you actually need MdeModulePkg.

Reply:
Also appear to not need MsCorePkg, removing both.